### PR TITLE
feat(codewhisperer): do not format code on acceptance

### PR DESCRIPTION
**Pull request by zixlin7**
## Problem
we received user feedback on not wanting CodeWhisperer to format their documents
## Solution
don't automatically format user's code on acceptance

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .changes/next-release/Bug Fix-10942e31-b4d4-4eb2-8b12-4c792a831e98.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "CodeWhisperer won't automatically format code after acceping code suggestions"
```
Comment (leigaol) [inline .changes/next-release/Bug Fix-10942e31-b4d4-4eb2-8b12-4c792a831e98.json:3]: It is a change of product behavior, not necessarily a bug fix. Fix typo.

File: .changes/next-release/Feature-e19735b8-3902-4ace-826f-5809bb6e8b7d.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "CodeWhisperer won't automatically format code after accepting code suggestions"
```
Comment (andrewyuq) [inline .changes/next-release/Feature-e19735b8-3902-4ace-826f-5809bb6e8b7d.json:3]: `will no longer`?

File: src/codewhisperer/commands/onInlineAcceptance.ts [hunk 3]
```diff
@@ -83,13 +81,12 @@ export async function onInlineAcceptance(
         const languageContext = runtimeLanguageContext.getLanguageContext(acceptanceEntry.editor.document.languageId)
         const start = acceptanceEntry.range.start
         const end = acceptanceEntry.editor.selection.active
-        const languageId = acceptanceEntry.editor.document.languageId
         RecommendationHandler.instance.reportUserDecisionOfRecommendation(
             acceptanceEntry.editor,
             acceptanceEntry.acceptIndex
         )
         // codewhisperer will be doing editing while formatting.
-        // formatting should not trigger consoals auto trigger
```
Comment (andrewyuq) [inline src/codewhisperer/commands/onInlineAcceptance.ts:92]: :)

Comment (andrewyuq) [reply src/codewhisperer/commands/onInlineAcceptance.ts:92]: We can also just remove this comment

Comment (zixlin7) [reply src/codewhisperer/commands/onInlineAcceptance.ts:92]: haha that's true will remove

**Review by andrewyuq — APPROVED** [review]

**Pull request merged**